code/game/machinery/autolathe_datums.dm

```diff
@@ -290,6 +290,17 @@
 	path = /obj/item/ammo_magazine/c38/rubber
 	category = "Arms and Ammunition"
 
+/datum/autolathe/recipe/magazine_543
+	name = "ammunition (5.43mm)"
```

Comment:
This is a very snowflake ammo caliber. Why not just nerf 5.56 slightly, buff 7.62 and use 556?